### ISSUE TITLE
doLayout?

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#264]
Background: in Dijit, ContentPane defaults to doLayout:true, which says that if
the content of the ContentPane is a single Dijit widget, then force the size of
that widget to equal the size of the ContentPane. This is a nice feature for
scenarios such as adding a Tree widget as the sole child of a left-side
ContentPane.

Adam added doLayout:false to ContentPanes in dijit.json when fixing bug 5138.
Is this the right approach? Seems a bit scary to go against Dijit's defaults.
Questions:
- Continue to force doLayout:false on ContentPanes that are created in Maqetta?
- What happens with inline edit if you add new tabs to a TabContainer? Are the
  new ContentPanes set up with doLayout set to false or true?
- Jon suggested that maybe we could offer a dialog when creating any of the
  container widgets with a checkbox for doLayout
- Adam has a better suggestion that we prompt the user upon dropping the first
  dijit into a ContentPane, with sticky bit to remember decision

------- Comment #1 From Adam Peller 2010-12-09 15:07 -------

While I agree with the rest of the narrative, I'm not sure there's anything
scary about going against Dijit's defaults. This is a public property meant to
be modified.  These widgets (ContentPane, also Stack and Tab Container)
basically have two modes of operation, so somehow explaining this and exposing
this to the user might be our only way out.
